cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/TicketGrantingTicketImpl.java

```diff
@@ -55,16 +57,22 @@
     private Boolean expired = Boolean.FALSE;
 
     /** Service that produced a proxy-granting ticket. */
-    @Column(name="PROXIED_BY", nullable=true)
+    @Lob
+    @Column(name="PROXIED_BY", nullable=true, length = Integer.MAX_VALUE)
     private Service proxiedBy;
 
     /** The services associated to this ticket. */
     @Lob
-    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = 1000000)
+    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = Integer.MAX_VALUE)
     private final HashMap<String, Service> services = new HashMap<>();
 
+    /** The PGTs associated to this ticket. */
+    @Lob
```

Comment:
> Prefer we don't do that, but I'll await your test results.

I agree, it's definitely a trade off. It makes the JPA implementation cleaner but at the cost of losing some cleanliness in the base model.

I was unable to find another solution that would allow us to leverage the already existing FK relationships.

here are the JPA changes I made: https://github.com/Unicon/cas/compare/slo-pgts...frett:slo-pgts-jpa

My preference would be to go this way because it make it easier to consistently maintain accurate state between TGTs & PGTs in the JPA model.